tests/test_tokens.py

```diff
@@ -281,6 +281,11 @@ def test_check_exp(self):
         with self.assertRaises(TokenError):
             token.check_exp('refresh_exp', current_time=current_time + timedelta(days=2))
 
+        # a token 1 day expired is valid if leeway is 2 days
+        token._token_backend.leeway = timedelta(days=2).total_seconds()
+        token.check_exp('refresh_exp', current_time=current_time + timedelta(days=1))
+        token._token_backend.leeway = 0
```

Comment:
what happens if this is removed?

Reply:
This could definitely be an own test case.